src/bench/mempool_stress.cpp

```diff
@@ -98,4 +100,27 @@ static void ComplexMemPool(benchmark::Bench& bench)
     });
 }
 
+static void MempoolCheck(benchmark::Bench& bench)
+{
+    FastRandomContext det_rand{true};
+    int childTxs = 2000;
+    if (bench.complexityN() > 1) {
+        childTxs = static_cast<int>(bench.complexityN());
+    }
+    std::vector<CTransactionRef> ordered_coins = CreateOrderedCoins(det_rand, childTxs, /* min_ancestors */ 5);
+    std::vector<size_t> random_index;
```

Comment:
This vector doesn't seem to be used anywhere after being filled (aka [WOM](https://en.m.wikipedia.org/wiki/Write-only_memory_(joke)))?

Reply:
oops :sweat_smile: was originally going to use this to evict transactions in random order, will remove